.gitignore

```diff
@@ -1,13 +1,38 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
```

Comment:
We should only ignore the files from package manager supported in repo, adding other manager files can obsuscate an user issue.